theme/styles/snippets/email-subscribe.scss

```diff
@@ -1,18 +1,18 @@
 .EmailSubscribe {
   @extend %transition;
   position: relative;
-  height: 42px;
-  top: -8px;
+  height: 1.911875rem;
+  top: -0.2rem;
```

Comment:
Button sizes had to change a bit because I realized the designs for the mobile footer use a smaller font than the `HomeHeader` (originally they were the same)